invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/uploads_items/ComputerTabletUploadsItem.js

```diff
@@ -0,0 +1,120 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2022 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import _get from "lodash/get";
+import _truncate from "lodash/truncate";
+import React from "react";
+import { Button, Icon, Item, Label } from "semantic-ui-react";
+import { SearchItemCreators } from "../../utils";
+
+export const ComputerTabletUploadsItem = ({
+  result,
+  index,
+  editRecord,
+  statuses,
+}) => {
+  const access_status_id = _get(result, "ui.access_status.id", "open");
+  const access_status = _get(result, "ui.access_status.title_l10n", "Open");
+  const access_status_icon = _get(result, "ui.access_status.icon", "unlock");
+  const creators = _get(result, "ui.creators.creators", []).slice(0, 3);
+  const subjects = _get(result, "ui.subjects", []);
+
+  const createdDate = result.ui?.created_date_l10n_long;
+  const description_stripped = result.ui?.description_stripped;
+  const publicationDate = result.ui?.publication_date_l10n_long;
+  const resource_type = result.ui?.resource_type?.title_l10n;
+  const title = result.metadata?.title;
+  const version = result.ui?.version;
+  const is_published = result.is_published;
+
+  const viewLink = is_published
+    ? `/records/${result.id}`
+    : `/uploads/${result.id}`;
+  const icon = is_published ? (
+    <Icon name="check" color="green" />
+  ) : (
+    <Icon name="upload" color="red" />
+  );
+
+  return (
+    <Item
+      key={index}
+      className="deposits-list-item mb-20 computer tablet only flex"
+    >
+      <div className="status-icon mr-10">
+        <Item.Content verticalAlign="top">
+          <Item.Extra>{icon}</Item.Extra>
+        </Item.Content>
+      </div>
+      <Item.Content style={{ cursor: "default" }}>
+        <Item.Extra>
+          {result.status in statuses && result.status !== "published" && (
+            <Label size="tiny" color={statuses[result.status].color}>
+              {statuses[result.status].title}
+            </Label>
+          )}
+          <Label size="tiny" color="blue">
+            {publicationDate && publicationDate} ({version && version})
```

Comment:
If `publicationDate === undefined` && `version === undefined`, would there be an empty `Label` component?